.github/workflows/test.yml

```diff
@@ -0,0 +1,45 @@
+name: Tests / Backend
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  postgres-container:
+    runs-on: ubuntu-latest
+    env:
+      PGUSER: postgres
+      PGPASSWORD: password
```

Comment:
wasn't sure how secrets worked with forked repos but we can probably figure this out and move this to a secret